Frameworks/UIKit/UIPanGestureRecognizer.mm

```diff
@@ -478,6 +478,7 @@ - (_UIPanGestureStage)_stage {
 */
 - (void)reset {
     _priv->touches.clear();
+	_priv->currentTranslation.x = _priv->currentTranslation.y = 0.f;
```

Comment:
if you don't mind, that would be great.  

---
In reply to: [78601747](https://github.com/Microsoft/WinObjC/pull/985#discussion_r78601747) [](ancestors = 78601747)